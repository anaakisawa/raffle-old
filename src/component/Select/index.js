import React, { Component } from 'react';

class Select extends Component {
  render () {
    return (
      <div className="custom-select">
        <select>
          {this.props.options.map((op) => (
            <option key={op} value={op}>
              {op}
            </option>
          ))}
        </select>
      </div>
    );
  }
}

export default Select;
