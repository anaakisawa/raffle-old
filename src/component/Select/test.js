import React from 'react';
import Select from './index';
import renderer from 'react-test-renderer';

describe('Component - Select', () => {
  test('renders a dropdown element', () => {
    const options = ['auto','vip','tecnologia'];
    const component = renderer.create(
      <Select options={options}/>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});
