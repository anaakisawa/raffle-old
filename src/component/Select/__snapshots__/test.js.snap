// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Select renders a dropdown element 1`] = `
<div
  className="custom-select"
>
  <select>
    <option
      value="auto"
    >
      auto
    </option>
    <option
      value="vip"
    >
      vip
    </option>
    <option
      value="tecnologia"
    >
      tecnologia
    </option>
  </select>
</div>
`;
